configs/paths.go

```diff
@@ -34,9 +34,9 @@ const (
 	BitriseSourceDirEnvKey = "BITRISE_SOURCE_DIR"
 	// BitriseDeployDirEnvKey ...
 	BitriseDeployDirEnvKey = "BITRISE_DEPLOY_DIR"
-	// BitriseTestDeployDirEnvKey ...
+	// BitriseTestDeployDirEnvKey contains a path from Deploy to Bitrise.io step will read each test results (all subdirs / from all steps)
 	BitriseTestDeployDirEnvKey = "BITRISE_TEST_DEPLOY_DIR"
-	// BitriseTestResultDirEnvKey ...
+	// BitriseTestResultDirEnvKey contains a path where test results from a single step can be copied over and will be deployed
```

Comment:
`A unique subdirectory in BITRISE_TEST_DEPLOY_DIR for each step run, steps should place test reports and attachments into this directory`